models/endoscopic_tool_segmentation/configs/metadata.json

```diff
@@ -12,7 +14,7 @@
         "0.1.0": "complete the first version model package",
         "0.0.1": "initialize the model package structure"
     },
-    "monai_version": "1.0.1",
+    "monai_version": "1.2.0",
```

Comment:
should be "1.2.0rc2", as "1.2.0" is not existing

Reply:
Hi @binliunls , I just discussed with Nic, and we decided to use "1.2.0rc2" until next release is published.